cairo/tests/ethereum/cancun/vm/instructions/test_system_cairo.cairo

```diff
@@ -0,0 +1,32 @@
+from starkware.cairo.common.cairo_builtins import BitwiseBuiltin, KeccakBuiltin, PoseidonBuiltin
+from starkware.cairo.common.registers import get_label_location
+
+from ethereum.cancun.vm.instructions.system import generic_create
+from ethereum.cancun.vm.interpreter import process_create_message, process_message
+from ethereum_types.numeric import U256, Uint
+from ethereum.cancun.fork_types import Address
+from ethereum.cancun.vm.exceptions import EthereumException
+from ethereum.cancun.vm import Evm
+
+func test_generic_create{
```

Comment:
do we need to create this function because we can't get the label of `process_create_message` otherwise